test/unit/parser.test.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 const fs = require('fs');
-const Parser = require('markbind/src/parser');
-const VariablePreprocessor = require('markbind/src/preprocessors/variablePreprocessor');
+const Parser = require('markbind/src/Parser');
+const VariablePreprocessor = require('markbind/src/preprocessors/VariablePreprocessor');
```

Comment:
I'll be touching on `variablePreprocessor` shortly, don't mind if you want to include it here though

Reply:
Touching in what way?

Reply:
#1263, a bugfix + small directory reorganization for variable processing files. Plan to resume work on it after moving `packages/markbind` to minimise conflict resolution though